tests/atomistics/structure/test_neighbors.py

```diff
@@ -351,6 +351,20 @@ def test_chemical_symbols(self):
         neigh = vacancy.get_neighbors(num_neighbors=None, cutoff_radius=3)
         self.assertEqual(neigh.chemical_symbols[0,-1], 'v')
 
+    def test_steinhardt_parameters(self):
+        neigh = StructureFactory().ase_bulk('Al').get_neighbors(num_neighbors=12)
+        # values obtained with pyscal
+        self.assertAlmostEqual(0, neigh.get_steinhardt_parameter(2)[0])
+        self.assertAlmostEqual(0.19094065395649323, neigh.get_steinhardt_parameter(4)[0])
+        self.assertAlmostEqual(0.5745242597140696, neigh.get_steinhardt_parameter(6)[0])
+        neigh = StructureFactory().ase_bulk('Ti').get_neighbors(num_neighbors=12)
```

Comment:
Changing the hcp structure to Mg should make comparison to reported publications such as [this](https://doi.org/10.1063/1.4774084) easier. The slightly lower c/a ratio of Ti changes the value of Steinhardt parameters.

Reply:
Ah super nice! Thanks!